pkg/asset/imagebased/configimage/configiso.go

```diff
@@ -0,0 +1,134 @@
+package configimage
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	"github.com/openshift/assisted-image-service/pkg/isoeditor"
+	"github.com/openshift/installer/pkg/asset"
+	agentAsset "github.com/openshift/installer/pkg/asset/agent"
+)
+
+const (
+	configImageFilename = "imagebasedconfig.noarch.iso"
+	configImageLabel    = "imagebased_configimage"
```

Comment:
I think ibi expects the iso label too be cluster-config
https://github.com/openshift-kni/lifecycle-agent/blob/0bad066d0bffb23c1318ef4327d69e79b55964ff/api/seedreconfig/seedreconfig.go#L8

Reply:
Oh nice catch, I'll fix this :+1: